db/seeds.rb

```diff
@@ -5,3 +5,14 @@
+providers.each { |provider_attributes| Provider.create!(provider_attributes) }
+
+services = [{:provider_name=>"12th & Park Shelter", :category=>"Housing", :name=>"Emergency Shelter", :male=>true, :female=>false},{:provider_name=>"Gateway 180", :category=>"Housing", :name=>"Emergency Shelter", :male=>false, :female=>true},{:provider_name=>"Loaves & Fishes Inc.", :category=>"Housing", :name=>"Emergency Shelter", :male=>false, :female=>true},{:provider_name=>"Our Lady's Inn", :category=>"Housing", :name=>"Emergency Shelter", :male=>false, :female=>"TRUE (if pregnant)"},{:provider_name=>"Peter & Paul", :category=>"Housing", :name=>"Emergency Shelter", :male=>true, :female=>false},{:provider_name=>"Room at The Inn", :category=>"Housing", :name=>"Emergency Shelter", :male=>false, :female=>true},{:provider_name=>"Saint Patrick Center- Women's Night", :category=>"Housing", :name=>"Emergency Shelter", :male=>false, :female=>true},{:provider_name=>"Salvation Army Family Haven", :category=>"Housing", :name=>"Emergency Shelter", :male=>true, :female=>true},{:provider_name=>"BIDDLE HOUSE OPPORTUNITIES CENTER", :category=>"Food", :name=>"Food Programs & Pantries", :male=>false, :female=>false},{:provider_name=>"SHAMROCK CLUB AT ST. PATRICK'S CENTER", :category=>"Food", :name=>"Food Programs & Pantries", :male=>false, :female=>false},{:provider_name=>"NEW LIFE EVANGELISTIC CENTER", :category=>"Food", :name=>"Food Programs & Pantries", :male=>false, :female=>false},{:provider_name=>"CHRIST CHURCH CATHEDRAL", :category=>"Food", :name=>"Food Programs & Pantries", :male=>false, :female=>false},{:provider_name=>"FATHER DEMPSEY'S HOME", :category=>"Food", :name=>"Food Programs & Pantries", :male=>false, :female=>false},{:provider_name=>"ENERGY CARE", :category=>"Utilities", :name=>"Utility Assistance", :male=>false, :female=>false},{:provider_name=>"FRANCISCAN CONNECTION", :category=>"Utilities", :name=>"Utility Assistance", :male=>false, :female=>false},{:provider_name=>"PAKT Community Resource Center", :category=>"Utilities", :name=>"Utility Assistance", :male=>false, :female=>false}]
+services.each do |service_attributes|
```

Comment:
Fixed in https://github.com/tpitale/counselor/commit/b47e8661555bdb751706ae7e772fbda325c05049